templates/aws/aws_cloudformation_template.yml

```diff
@@ -23,30 +35,50 @@ Resources:
   INSTANCE:
     Type: AWS::EC2::Instance
     Properties:
-      ImageID: !Ref AMI_ID
+      ImageId: !Ref AMIID
       InstanceType: "m5a.xlarge"
       KeyName: "RobG_id_rsa"
+      SubnetId: !Ref SUBNETIDFORVM
+      IamInstanceProfile: !Ref IAMROLEFORVM 
+      PrivateDnsNameOptions:
+          EnableResourceNameDnsARecord: true
+      SecurityGroupIds:
+        - !Ref SECURITYGROUPID 
       PropagateTagsToVolumeOnCreation: true
       Tags: 
         - Key: "LansaVersion"
-          Value: !Ref LANSA_VERSION
+          Value: !Ref LANSAVERSION
         - Key: "Name"
           Value: "DB Regression Test"        
   
-  MYSQL_RDS:
+  MYSQLRDS:
     Type: AWS::RDS::DBInstance
     Properties:
-      DBSnapshotIdentifier: !Ref MYSQL_SNAPSHOT_ARN
+      DBInstanceIdentifier:  !Join [ "", [ "mysql", !Ref LANSAVERSION ] ] 
+      DBSnapshotIdentifier: !Ref MYSQLSNAPSHOTARN
       AllocatedStorage: '50'
       DBInstanceClass: db.t3.large
       Engine: mysql
+      VPCSecurityGroups:
+        - !Ref SECURITYGROUPID
+      EnablePerformanceInsights: true
       CopyTagsToSnapshot: true
+      Tags: 
+        - Key: "LansaVersion"
+          Value: !Ref LANSAVERSION
 
-  ORACLE_RDS:
+  ORACLERDS:
     Type: AWS::RDS::DBInstance
     Properties:
-      DBSnapshotIdentifier: !Ref ORACLE_SNAPSHOT_ARN
+      DBInstanceIdentifier: !Join [ "", [ "ora", !Ref LANSAVERSION ] ]
```

Comment:
Good. As specified.

Reply:
You have not answered any of the comments I made previously. You need to do that. We stated this in a meeting.

Reply:
The name of the stack is not unique. DB-Regression-[LansaVersion]

It should not have "STACK" in it.

If you just tag the stack with LansaVersion, all the resources will automatically be tagged. Remove all the individual resource tags

Reply:
I've left the stack for you to delete in case you need to refer to it given I've commented on it.

Reply:
I have implemented the previous requested change of adding the tags to RDS.